Index: kitty/terminfo/kitty.terminfo
===================================================================
--- kitty.orig/terminfo/kitty.terminfo	2019-01-12 19:48:53.646636866 +0700
+++ kitty/terminfo/kitty.terminfo	2019-01-12 19:48:53.642636969 +0700
@@ -221,6 +221,8 @@
 	setaf=\E[%?%p1%{8}%<%t3%p1%d%e%p1%{16}%<%t9%p1%{8}%-%d%e38;5;%p1%d%;m,
 	setrgbb=\E[48:2:%p1%d:%p2%d:%p3%dm,
 	setrgbf=\E[38:2:%p1%d:%p2%d:%p3%dm,
+	setb24=\E[48:2:%p1%{65536}%/%d:%p1%{256}%/%{255}%&%d:%p1%{255}%&%dm,
+	setf24=\E[38:2:%p1%{65536}%/%d:%p1%{256}%/%{255}%&%d:%p1%{255}%&%dm,
 	sgr=%?%p9%t\E(0%e\E(B%;\E[0%?%p6%t;1%;%?%p2%t;4%;%?%p1%p3%|%t;7%;%?%p4%t;5%;%?%p7%t;8%;m,
 	sgr0=\E(B\E[m,
 	sitm=\E[3m,
